### PR TITLE
BCDA-7332: update PACA user guide

### PR DESCRIPTION
**Pull request by karinamzalez**
## 🎫 Ticket

https://jira.cms.gov/browse/BCDA-7332

## 🛠 Changes

- update PACA user guide 

## ℹ️ Context for reviewers

- new user guide should be linked from user guide page 

https://github.com/CMSgov/bcda-static-site/assets/14364118/ed59c4e7-bec7-4c13-afc6-0e99800b7ab6

## ✅ Acceptance Validation

- assert that the user can view: https://app.box.com/file/1282441965098?sb=/activity/versions/1401895878698 when attempting to navigate to user guide download link

## 🔒 Security Implications

- [ ] This PR adds a new software dependency or dependencies.
- [ ] This PR modifies or invalidates one or more of our security controls.
- [ ] This PR stores or transmits data that was not stored or transmitted before.
- [ ] This PR requires additional review of its security implications for other reasons.

If any security implications apply, add Jason Ashbaugh (GitHub username: StewGoin) as a reviewer and do not merge this PR without his approval.

### CODE REVIEW

File: _includes/partial/user_guide.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <div>
-  <p>The BCDA Partially Adjudicated Claims User Guide contains information found on this page and additional useful information for REACH ACOs. You can download the User Guide <a href="/assets/data/BCDA_Partially_Adjudicated_Claims_Enhancement_User_Guide_v1.0.0.pdf" target="blank" class="in-text__link">here</a>.
+  <p>The BCDA Partially Adjudicated Claims User Guide contains information found on this page and additional useful information for REACH ACOs. You can download the User Guide <a href="/assets/data/BCDA_Partially_Adjudicated_Claims_Enhancement_User_Guide_v1.2.0.pdf" target="blank" class="in-text__link">here</a>.
```
Comment (laurenkrugen-navapbc) [inline _includes/partial/user_guide.html:2]: I still get an error when attempting to open the new file; do you get the same thing when you run locally and is that expected behavior when running locally?

**Comment by laurenkrugen-navapbc** [comment]
I looked at the `Updates` page and it looks like `Version 1 of the BCDA Partially Adjudicated Claims Enhancement live for use on May 16!` is duplicated on there again - can you confirm the same?

File: _includes/partial/user_guide.html (hunk 1 above)
Comment (karinamzalez) [reply _includes/partial/user_guide.html:2]: @laurenkrugen-navapbc I updated the PR description to attach a screen recording of what I am seeing from my end.

**Comment by karinamzalez** [comment]
> I looked at the `Updates` page and it looks like `Version 1 of the BCDA Partially Adjudicated Claims Enhancement live for use on May 16!` is duplicated on there again - can you confirm the same?

I'm not seeing that from my end and no changes are being made to the updates page with this PR -- apart from the updated import for the user guide pdf 🤔

**Comment by laurenkrugen-navapbc** [comment]
> I looked at the `Updates` page and it looks like `Version 1 of the BCDA Partially Adjudicated Claims Enhancement live for use on May 16!` is duplicated on there again - can you confirm the same?

I had the old branch - apologies! new branch looks good.

**Review by laurenkrugen-navapbc — APPROVED** [review]

**Pull request merged**